helm/trento-ai-companion/Chart.yaml

```
@@ -0,0 +1,13 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: trento-ai-companion
+description: A Helm chart for deploying the Trento AI Companion application
```

just call it what it is :)
```suggestion
description: A Helm chart for deploying the Trento MCP Server component
```